GPU/GLES/TextureCacheGLES.cpp

```diff
@@ -885,37 +864,9 @@ void TextureCacheGLES::LoadTextureLevel(TexCacheEntry &entry, ReplacedTexture &r
 			WARN_LOG(G3D, "Got an error BEFORE texture upload: %08x (%s)", err, GLEnumToString(err).c_str());
 		}
 		if (IsFakeMipmapChange())
-			glTexImage2D(GL_TEXTURE_2D, 0, components, w, h, 0, components2, dstFmt, pixelData);
+			render_->TextureImage(entry.textureName, 0, w, h, components, components2, dstFmt, pixelData);
 		else
-			glTexImage2D(GL_TEXTURE_2D, level, components, w, h, 0, components2, dstFmt, pixelData);
-		if (!lowMemoryMode_) {
```

Comment:
I don't know how we'll best do that now...

Reply:
Added it on init in #10548.  It'll detect it a few frames behind, so might still have crashiness, but since we create all textures at once, I think we may be less likely to crash on OOM.  It'll show the osm message as before, if possible.

-[Unknown]